TF_Settings_Web/src/Components/Pages/Camera/CalibrationComponents.tsx

```diff
@@ -0,0 +1,178 @@
+import '../../../Styles/Camera/Calibrate.css';
+import 'react-circular-progressbar/dist/styles.css';
+
+import { CreateTypes } from 'canvas-confetti';
+import React, { useRef } from 'react';
+import { CSSProperties } from 'react';
+import ReactCanvasConfetti from 'react-canvas-confetti';
+import { buildStyles, CircularProgressbar } from 'react-circular-progressbar';
+import { CSSTransition, SwitchTransition } from 'react-transition-group';
+
+import FingerprintIcon from '../../../Images/Camera/Fingerprint_Icon.svg';
+import IconTextButton from '../../Controls/IconTextButton';
+
+interface CalibrateInstructionsProps {
+    progress: number;
+    containerStyle: CSSProperties;
+}
+export const CalibrateInstructions: React.FC<CalibrateInstructionsProps> = ({ progress, containerStyle }) => {
+    const instructionsText = (
+        <h1>
+            Hold INDEX FINGER against <br /> this <span style={{ color: '#01EB85' }}>GREEN CIRCLE</span>
```

Comment:
Could we move the colour into a CSS class? It looks like it's being used in a few places and that way we could update it everywhere easily